sorc/global_cycle.fd/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@
 set(lib_src
     machine.f90
     num_parthds.f90
-    sfcsub.F
+    ../../ccpp-physics/physics/sfcsub.F
```

Comment:
@GeorgeGayno-NOAA When I tried your ccpp-physics branch, ufs_utils failed to compile because this file ../../ccpp-physics/physics/sfcsub.F does not exist.  For global-workflow, this file can be linked to ../../../ufs_model.fd/FV3/ccpp/physics/physics/sfcsub.F; otherwise, you need to check out this file from CCPP github repo.

Reply:
Try 'git clone --recursive'. 

Or 'git submodule init', then 'git submodule update'

Reply:
got it. thanks

Reply:
Thanks for reviewing. I can't add fractional grid support for global_cycle (#549) until this PR is merged.